views/drones/list.hbs

```diff
@@ -1 +1,18 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<h2>Available drones</h2>
+<div>
+    {{#each drones}}
+    <div>
+        <h3>{{this.name}}</h3>
+        <li>Propellers: {{this.propellers}} </li>
```

Comment:
You should always have either `<ul>` or `<ol>` surrounding `<li>` elements (doing otherwise might work/render but it's semantically incorrect).